proposal/package-importer.d.ts.md

```diff
@@ -504,6 +504,16 @@ potential subpaths, resolving for partials and file extensions.
 
 ## Embedded Protocol
 
+An Importer that resolves `pkg:` URLs using the [node resolution algorithm]. It
+is instantiated with an associated `entry_point_url` set to the value of
+`file://${process.cwd()}/`.
```

Comment:
This is confusing—if the `entry_point_url` is a field of the proto, isn't it up to the host to provide it rather than the spec?

Reply:
Ah, yes. Does this need to be specified elsewhere for the host, or is that not part of the spec?

Reply:
The JS host isn't part of the spec, except in that it has to follow the JS API spec.